### ISSUE TITLE
Overall Dedupe v2 table color scheme

### ISSUE THREAD

**Issue by DavidNesbitt2020** [pepfar-datim/datim-dedupe#149]
Related to #139 and #147 

When looking at multiple dedupes (screenshot below), it can be hard to differentiate between various duplications being analyzed and updated. I think the grayscale background coloring makes the Duplicated column difficult to read.

I also think this screenshot lends more credence to having the descriptive headers mentioned in #142 

![image](https://user-images.githubusercontent.com/73136568/101206990-ed798080-363d-11eb-8cb5-de636d4bb564.png)

@benguaraldi for MVP assessment

**Comment by benguaraldi** [comment]
@DavidNesbitt2020 Is this a duplicate of #144?

**Comment by DavidNesbitt2020** [comment]
@benguaraldi Maybe?  Will the changes in #144 also update the other table colors and shading, not just the Status column?

**Comment by benguaraldi** [comment]
Yes, they will.

After #144 (which isn't MVP), let's re-evaluate and decide whether there's anything to do with this ticket.
